### ISSUE TITLE
Update for Wonderlands

### ISSUE THREAD

**Issue by bmfrosty** [matt1484/bl3_auto_vip#42]
With the release coming up, I suspect it will use similar shift codes.

**Comment by squatto** [comment]
I came back to this repo to see if it would work with Wonderlands codes. It uses the same Shift-based codes, so they should work. I'm going to give it a whirl right now and see if it still works (and attempt to fix it if it doesn't!)

**Comment by squatto** [comment]
Nope, it definitely isn't working anymore. Discussion on why is here: #28

**Comment by squatto** [comment]
I fixed the auth problems in PR #43. I'm adding Wonderlands redemption now - I'll update this issue when I've submitted the changes 👍🏻

**Comment by bmfrosty** [comment]
It's been forked here - https://github.com/jauderho/bl3auto/

This is a working version and does support support Wonderlands shift codes.

**Comment by squatto** [comment]
Oh perfect, I had no idea! Thanks for sending that 👍🏻
